app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -105,6 +105,7 @@ public void run() {
     static final int GestureSwipeLeft = 0;
     static final int GestureSwipeRight = 1;
     static final int SwipeDelay = 1000; // milliseconds
+    private static final long MIN_RESTART_INTERVAL_MS = 3000;
```

Comment:
I wonder if this shouldn't be public somewhere so it is only define in one place?

Reply:
I reused the variable already defined in CrashReporterService